xml/security_ldap.xml

```diff
@@ -889,7 +663,258 @@ telephoneNumber: +49 1234 567-10</screen>
+       <imagedata fileref="yast2_auth_client_ldap.png" width="65%" format="PNG"/>
+      </imageobject>
+      <imageobject role="fo">
+       <imagedata fileref="yast2_auth_client_ldap.png" width="65%" format="PNG"/>
+      </imageobject>
+     </mediaobject>
+    </informalfigure>
+   </step>
+   <step>
+    <para>
+     Specify one or more LDAP server URLs, host names, or IP addresses under
+     <guimenu>Enter LDAP server locations</guimenu>. When specifying multiple
+     addresses, separate them with spaces.
+    </para>
+   </step>
+   <step>
+    <para>
+     Specify the appropriate LDAP distinguished name (DN) under
+     <guimenu>DN of Search Base</guimenu>. For example, a valid entry could
+     be <literal>dc=example,dc=com</literal>.
+    </para>
+   </step>
+   <step>
+    <para>
+     If your LDAP server supports TLS encryption, choose the appropriate
+     security option under <guimenu>Secure LDAP Connection</guimenu>.
+    </para>
+    <para>
+     To first ask the server whether it supports TLS encryption and be able
+     to downgrade to an unencrypted connection if it does not, use
+     <guimenu>Secure Communication via StartTLS</guimenu>.
+    </para>
+   </step>
+   <step>
+    <para>
+     Activate other options as necessary:
+    </para>
+    <itemizedlist>
+     <listitem>
+      <para>
+       You can <guimenu>Allow users to authenticate via LDAP</guimenu> and
+       <guimenu>Automatically Create Home Directories</guimenu> on the local
+       computer for them.
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       Use <guimenu>Cache LDAP Entries For Faster Response</guimenu> to cache
+       LDAP entries locally. However, this bears the danger that entries can
+       be slightly out of date.
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       Specify the types of data that should be used from the LDAP source,
+       such as
+       <guimenu>Users</guimenu> and <guimenu>Groups</guimenu>,
+       <guimenu>Super-User Commands</guimenu>, and <guimenu>Network Disk
+       Locations</guimenu> (network-shared drives that can be automatically
+       mounted on request).
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       Specify the distinguished name (DN) and password of the user under
+       whose name you want to bind to the LDAP directory in <guimenu>DN of
+       Bind User</guimenu> and <guimenu>Password of the Bind User</guimenu>.
+      </para>
+      <para>
+       Otherwise, if the server supports it, you can also leave both text
+       boxes empty to bind anonymously to the server.
+      </para>
+      <warning>
+       <title>Authentication Without Encryption</title>
+       <para>
+        When using authentication without enabling transport encryption
+        using TLS or StartTLS, the password will be transmitted in the clear.
+       </para>
+       <remark>
+        Is the YaST dialog compatible with SASL in any way? - sknorr,
```

Comment:
SASL is too complicated for a setup like this, not worth including.

Reply:
Turned remark into a comment and added your answer, just for the records.